Lib9c/Delegation/Delegation.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Numerics;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+
+namespace Nekoyume.Delegation
+{
+    public class Delegation
+    {
+        public Delegation(
+            Bond bond = null,
+            UnbondLockIn unbondLockIn = null,
+            RebondGrace rebondGrace = null,
+            UnbondingSet unbondingSet = null)
+        {
+            Bond = bond;
+            UnbondLockIn = unbondLockIn;
+            RebondGrace = rebondGrace;
+            UnbondingSet = unbondingSet;
+        }
+
+        public Bond Bond { get; }
+
+        public UnbondLockIn UnbondLockIn { get; }
+
+        public RebondGrace RebondGrace { get; }
+
+        public UnbondingSet UnbondingSet { get; }
+
+        public FungibleAssetValue? IncompleteBond { get; private set; }
+
+        public FungibleAssetValue? IncompleteUnbond { get; private set; }
+
+        public void AddBond(FungibleAssetValue fav, BigInteger share)
```

Comment:
If this method is only used in the `Delegatee` class, I think that access-modifier of this method must be `internal`.

Reply:
👍